include/circt/Dialect/FIRRTL/FIRRTLStatements.td

```diff
@@ -272,6 +272,27 @@ def MatchOp : FIRRTLOp<"match", [SingleBlock, NoTerminator,
   }];
 }
 
+def PropAssignOp : FIRRTLOp<"passign", [FConnectLike, SameTypeOperands,
```

Comment:
Maybe call it `propassign` or `prop.assign` or something so it isn't confused with procedural assignment?

Reply:
Went for `propassign`.